xbmc/cores/DllLoader/DllLoader.cpp

```diff
@@ -213,7 +213,7 @@ void DllLoader::PrintImportTable(ImportDirTable_t *ImportDirTable)
   ImportDirTable_t *Imp = ImportDirTable;
   int HavePrinted = 0;
 
-  CLog::Log(LOGDEBUG, "The Coff Image contains the following imports:\n\n");
+  CLog::Log(LOGDEBUG, "The Coff Image contains the following imports:\n");
```

Comment:
Yes, I guess that might have been the intention, but as we know that's now going to result in a blank line appearing in the log, which is going to be a little weird. If separation between log messages is required then shouldn't it log a blank message?